src/hotkeys_provider.rs

```diff
@@ -34,14 +35,13 @@ pub struct HotkeysContext {
 }
 
 pub fn provide_hotkeys_context<T>(
+    node_ref: NodeRef<T>,
```

Comment:
I didn't tested the code, but... is this `NodeRef<ElementDescriptor + 'static + Clone>` really reactive? I think that not because no signal types are implementing `ElementDescriptor`. You would probably want to create an example of a `Signal` (or even better, an `Into<Signal>`) returning a `NodeRef` to make sure.

If not, consider to use the defintion proposed at https://github.com/gaucho-labs/leptos-hotkeys/pull/45#discussion_r1514968694

Reply:
This is still relevant, but I'm hiding it to move forward this PR.